src/index.ts

```diff
@@ -2,6 +2,7 @@ import slugify from '@sindresorhus/slugify';
 import { Step } from './base';
 import { Conditional } from './conditional';
 import { KeyValue, KeyValueImpl } from './key_value';
+import { MutatorFn } from './serializers';
```

Comment:
- [ ] we need to export this as well, so the type can be used by people using the package later on.

Reply:
done